protocol/protocol.proto

```diff
@@ -627,11 +627,21 @@ message BlockchainTransaction {
         Event event = 1; // event emitted by the tipping facet
         bytes toUserAddress = 2; // user that received funds
     }
+
+    message Transfer {
+        bytes address = 1;
+        string amount = 2; // uint64 isn't big enough
+        bytes sender = 3;
+        bytes message_id = 4;
+        bytes channel_id = 5;
```

Comment:
should we add a flag here for buy/sell mapping to the to/from the sender?

Reply:
something like:
enum TransferType {
    fromSender,
    toSender,
}
and validate here
```
if transfer.From.Cmp(senderAddress) != 0 && transfer.To.Cmp(senderAddress) != 0 {
				continue
			}
```

Reply:
idk if enum is better, it's late here lol. bool is great!